server/data/ReportedAdjudicationResult.ts

```diff
@@ -48,7 +48,7 @@ export interface ReportedAdjudicationEnhanced extends ReportedAdjudication {
 export type ReportedAdjudicationFilter = {
   fromDate: moment.Moment
   toDate: moment.Moment
-  status?: ReportedAdjudicationStatus
+  status: ReportedAdjudicationStatus[]
```

Comment:
I think this needs to be 
`status: ReportedAdjudicationStatus | ReportedAdjudicationStatus[]`
I've been through and logged the statuses, trying different combinations - when there's only one status sent, it's not sent as an array but a string. So the two alternatives are needed here

Reply:
I think all works with the status's as an array , but this may explain why I still had a TODO on the line
`${(filter.status && `&status=${filter.status}`) || `&status=${allStatuses}`}` // TODO
As i above &status=${allStatuses}` is not an array 
Happy put your suggestion back in - perhaps we should revisit this at a later date.